src/pystack/_pystack/pytypes.cpp

```diff
@@ -503,7 +503,7 @@ std::string
 GenericObject::toString(ssize_t max_size) const
 {
     std::stringstream os;
-    os << "<" << std::hex << std::showbase << d_classname << " at " << d_addr << ">";
+    os << "<" << std::hex << d_classname << " at 0x" << d_addr << ">";
```

Comment:
This change looks dumb, but it's because `std::showbase` doesn't show `0x` on 0, and `<invalid object at 0>` is more likely to confuse users than `<invalid object at 0x0>` because the latter is recognizable as an address.